app/views/shared/_show_breadcrumbs.html.erb

```diff
@@ -1,20 +1,29 @@
 <% parents = Arclight::Parents.from_solr_document(document).as_parents %>
+<% h1_index = 0 %>
 <nav aria-label="breadcrumb">
   <ol class="breadcrumb">
-    <li class="breadcrumb-item al-bc-item breadcrumb-item-1">
+    <li class="media breadcrumb-item  breadcrumb-item-1">
       <% if document.repository_config.present? %>
-        <%= link_to(document.repository_config.name, arclight_engine.repository_path(document.repository_config.slug)) %>
+        <span class="media-body"><%= blacklight_icon :repository, classes: 'al-repository-content-icon' %></span>
+        <span class="col"><%= link_to(document.repository_config.name, arclight_engine.repository_path(document.repository_config.slug)) %></span>
       <% else %>
-          <span><%= t('arclight.show_breadcrumb_label') %></span>
+          <span><%= blacklight_icon :repository, classes: 'al-repository-content-icon' %></span><span class="col"><%= t('arclight.show_breadcrumb_label') %></span>
       <% end %>
     </li>
     <% parents.each_with_index.map do |parent, index| %>
-      <%= content_tag :li, class: "breadcrumb-item breadcrumb-item-#{index + 2} al-bc-item" do %>
-        <%= link_to parent.label, solr_document_path(parent.global_id) %>
-      <% end %>
+      <%= content_tag :li, class: "breadcrumb-item breadcrumb-item-#{index + 1} media" do %>
+        <% bc_icon = parent.level.downcase %>
```

Comment:
Could we use `document_header_icon` here? Or even generalize the name?

Reply:
FIXED: renamed to "document_or_parent_icon" and handles cases previously shown here.

@labradford, I added 'aria-hidden="true"' to the icon spans. :-)